File: RoadsideAssistanceInfoHub/src/Vehicles/Vehicles.Core/Models/Vehicle.cs
namespace Vehicles.Core.Models;

public class Vehicle
{
    public Guid VehicleId { get; set; }
    public Guid TenantId { get; set; }
    public string Make { get; set; } = string.Empty;
    public string Model { get; set; } = string.Empty;
    public int Year { get; set; }
    public string? VIN { get; set; }
    public string? LicensePlate { get; set; }
    public string? Color { get; set; }
    public decimal? CurrentMileage { get; set; }
    public string? OwnerName { get; set; }
    public string? Notes { get; set; }
    public bool IsActive { get; set; } = true;
}

File: VehicleValueTracker/src/VehicleValueTracker.WebApp/projects/vehicle-value-tracker-shared/src/lib/models/vehicle.ts
export interface Vehicle {
  vehicleId: string;
  make: string;
  model: string;
  year: number;
  trim?: string;
  vin?: string;
  currentMileage: number;
  purchasePrice?: number;
  purchaseDate?: string;
  color?: string;
  interiorType?: string;
  engineType?: string;
  transmission?: string;
  isCurrentlyOwned: boolean;
  notes?: string;
}
